ghcide/bench/example/HLS

```diff
@@ -1 +0,0 @@
-../../..
```

Comment:
This was an accident, undo!

Reply:
Apparently, it should not have ever been committed, and CI is still happy, so keep deleted